internal/validations/clusterlogforwarder/validate_name.go

```diff
@@ -0,0 +1,16 @@
+package clusterlogforwarder
+
+import (
+	v1 "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+	"github.com/openshift/cluster-logging-operator/internal/validations/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func validateName(clf v1.ClusterLogForwarder, k8sClient client.Client, extras map[string]bool) (error, *v1.ClusterLogForwarderStatus) {
```

Comment:
Should validateName be added to https://github.com/openshift/cluster-logging-operator/blob/de1e4b3795f56092c3bb2df9f90639e7a8fca3f7/internal/validations/clusterlogforwarder/validations.go#L22 ?